pages/1.13/release-notes/1.13.1/index.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.1
+title: Release notes for 1.13.1
+menuWeight: 1
+excerpt: Release notes for DC/OS 1.13.1, including Open Source attribution, and version policy.
+---
+DC/OS 1.13.1 was released on May 31, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.13.1/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="http://downloads.mesosphere.com/dcos-enterprise/stable/1.13.1/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the <a href="https://support.mesosphere.com/s/downloads">support website</a>. For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
```

Comment:
Hi @lsgunnlsgunn  - I think, it might be a good idea to *hightlight* that this is the release that we recommend our customers should upgrade to from 1.12.

This should come very early in the Release summary section, and we cloud also add a section like

> New Features and Capabilities in DC/OS 1.13.0 and DC/OS 1.13.1

For all practical purposes, our intention is to guide our customers start using DC/OS 1.13 from DC/OS 1.13.1 version.

HTH

Reply:
That's an interesting point. I can copy the new features part from 1.13.0. I'll try to come up with something diplomatic to say about moving directly to this patch. Thanks for the suggestions!

Reply:
Please give a pointer to the advisory on upgrade https://support.mesosphere.com/s/article/Known-Issue-Marathon-MSPH-2019-0004

Reply:
added.